configure.ac

```diff
@@ -438,6 +438,12 @@ AC_TYPE_UINT8_T
 pkgconfigdir=${libdir}/pkgconfig
 AC_SUBST(pkgconfigdir)
 
+#Add flags for SDL signoff called SDL325 - Compile With Defenses Enabled
+SDL_CFLAGS="-D_FORTIFY_SOURCE=2 -fPIE -fPIC -fstack-protector"
+SDL_LDFLAGS="-z noexecstack -z relro -z now -pie"
+AC_SUBST(SDL_CFLAGS)
+AC_SUBST(SDL_LDFLAGS)
+
```

Comment:
It would be better to just AC_SUBST the AM_CPPFLAGS and AM_LDFLAGS.  This way you don't have to modify any makefiles... all makefile targets will capture the flags automatically through AM_ flags (e.g. there are other makefiles/targets that you missed).

Reply:
As your requirement, I add these options into AM_flags, please review, thanks.